endpoints/openrtb2/video_auction.go

```diff
@@ -238,7 +238,7 @@ func cleanupVideoBidRequest(videoReq *openrtb_ext.BidRequestVideo, podErrors []P
 	return videoReq
 }
 
-func handleError(labels pbsmetrics.Labels, w http.ResponseWriter, errL []error, ao analytics.AuctionObject) {
+func handleError(labels *pbsmetrics.Labels, w *http.ResponseWriter, errL []error, ao *analytics.AuctionObject) {
```

Comment:
For context, the labels for handling an error case were not being updated due to the `handleError` function not using a pointer. Every request was then being recorded as OK. The http writer and analytics objects were also changed to pointers. The `ao` argument is recorded the same way as the labels so it was also missing updates. I am assuming the http writer should also be a pointer, but let me know if this should change.